### ISSUE TITLE
How to stop the daemon/forever

### ISSUE THREAD

**Issue by ttodua** [snail007/goproxy#546]
I havent used issue template, because I just have question (couldnt find answer in English):
how to stop the proxy if we had run it with `daemon` or `forever`.

`systemctl stop proxy` does not stop it: `Failed to stop proxy.service: Unit proxy.service not loaded.`

**Comment by snail007** [comment]
ps find pid and kill pid.

**Comment by ttodua** [comment]
yes, I have done that, but can you please add some built-in ways, so we could like:
`proxy --stop-all`  or `proxy --stop-daemons` or `proxy --stop-forever`
so it could simplify the process?
thanks

**Comment by ttodua** [comment]
@snail007 if you can add that above-mentioned feature? thanks
